styles/globals.css

```diff
@@ -4,4 +4,5 @@
 
 html {
   -webkit-font-smoothing: antialiased;
+  font-size: calc(100vmin / 720 * 9.3);
```

Comment:
The sizing looks slightly off compared to the current version:

<img width="1440" alt="Screen Shot 2023-02-09 at 9 56 42 AM" src="https://user-images.githubusercontent.com/1163252/217848342-b8754730-9771-478c-a0b2-5b1c891c6eaf.png">

With the changes:

<img width="1440" alt="Screen Shot 2023-02-09 at 9 56 47 AM" src="https://user-images.githubusercontent.com/1163252/217848402-76d2feb0-7cbb-4dd0-8ce0-4a5a6967c2f4.png">

Easier to see if you open both versions at 1080p:


https://user-images.githubusercontent.com/1163252/217848747-0d3e29a3-8b60-4744-bbd6-4e3950c06c22.mov

Reply:
I believe the base font size for Tailwind is 16px. With the responsive changes, I think we want to make sure the font-size at 1080p works out to be the same (16px)

Reply:
I think it should be fixed. Please take a look at the video:

https://user-images.githubusercontent.com/3447630/218055979-69deef07-fcfe-48e6-ba1a-0e0981284640.mov